Geminus City Tweaks/ Theta Project Change

lngredients opened the pull request:
- Changed name of eyeballs to Augmented Eyeballs in theta room.
- Changed name of Enhanced Heart to Augmented Heart in theta room.
- Expanded the Theta area to accommodate a new closet.
- Added in a Project Theta Closet.
- Added in Theta Project Armor, Helmet, Arm Guards, boots and undersuit. (All unique items with custom descriptions- All not as good as Spartan armor)
- Added in Augmented Lungs. Unique item comparable to Augmented Liver.
- Changed Max damage for Augmented Eyes to 60 from 45 to be slightly tougher as it doesn't offer that much of a benefit other then flash resistance.
- Adding in three stacks of ten sheets of glass into cargo storage area.
- Adding in two stacks of ten sheets of glass into the Engineering Area of Geminus.
- Added a Sink to the Padded Cell in Hospital.
- Added additional sink closer to Surgery table at GCPD Morgue.
- Moved Microwave in hospital break room so people can access the Getmore Vendor.
- Added in recharger at hospital where spare Defibs are kept.
- Added in sink to hospital Morgue.
- Added holopads inside a number of buildings outside of the slums. (10-15 extra Mostly in government/public buildings)
- Added sink to Viro Bathroom.

lngredients committed:
Geminus City Tweaks/ Theta Project Change

Changed name of eyeballs to Augmented Eyeballs in theta room.
Changed name of Enhanced Heart to Augmented Heart in theta room.
Expanded the Theta area to accommodate a new closet.
Added in a Project Theta Closet.
Added in Theta Project Armor, Helmet, Arm Guards, boots and undersuit. (All unique items with custom descriptions- All not as good as Spartan armor)
Added in Augmented Lungs. Unique item comparable to Augmented Liver.
Changed Max damage for Augmented Eyes to 60 from 45 to be slightly tougher as it doesn't offer that much of a benefit other then flash resistance.
Adding in three stacks of ten sheets of glass into cargo storage area.
Adding in two stacks of ten sheets of glass into the Engineering Area of Geminus.
Added a Sink to the Padded Cell in Hospital.
Added additional sink closer to Surgery table at GCPD Morgue.
Moved Microwave in hospital break room so people can access the Getmore Vendor.
Added in recharger at hospital where spare Defibs are kept.
Added in sink to hospital Morgue.
Added holopads inside a number of buildings outside of the slums. (10-15 extra Mostly in government/public buildings)
Added sink to Viro Bathroom.

DoctorOctopusTheOctopi reviewed (dismissed):
Hiya! I'll be peer-reviewing your stuff today. There is just a minor issue that can be resolved with a quick tutorial on github, but you're currently trying to push all of your changes into the main repository under a new area called geminus_city

https://gyazo.com/e3d56ea18a38fb85c24bf3fa2d13ad99

instead of this area right here

https://gyazo.com/28c657bc7347201123cba679e731fb90

If you need help rectifying this issue I'll be in discord my ckey is Octopus#9682

Thank you for your interest in developing with us boys at Halostation and we hope you stick around and stuff, because we need more devboys.

lngredients commented:
Alright I'll try to change it. Thanks.

lngredients committed:
Gem City Tweaks/Theta Project Additions

Changed name of eyeballs to Augmented Eyeballs in theta room.
Changed name of Enhanced Heart to Augmented Heart in theta room.
Expanded the Theta area to accommodate a new closet.
Added in a Project Theta Closet.
Added in Theta Project Armor, Helmet, Arm Guards, boots and undersuit. (All unique items with custom descriptions- All not as good as Spartan armor)
Added in Augmented Lungs. Unique item comparable to Augmented Liver.
Changed Max damage for Augmented Eyes to 60 from 45 to be slightly tougher as it doesn't offer that much of a benefit other then flash resistance.
Adding in three stacks of ten sheets of glass into cargo storage area.
Adding in two stacks of ten sheets of glass into the Engineering Area of Geminus.
Added a Sink to the Padded Cell in Hospital.
Added additional sink closer to Surgery table at GCPD Morgue.
Moved Microwave in hospital break room so people can access the Getmore Vendor.
Added in recharger at hospital where spare Defibs are kept.
Added in sink to hospital Morgue.
Added holopads inside a number of buildings outside of the slums. (10-15 extra Mostly in government/public buildings)
Added sink to Viro Bathroom.

lngredients committed:
Gem City/Theta Project Proper file placement

Moving files to the map folder

bloxgate commented:
Hi, I noticed that the PR description mentions several code changes, such as augmented lungs, however the changes list only shows the map changes.

Are you sure you committed everything?

lngredients commented:
So what I did was create a lung object, then directly edited the variables of that single object. Same with the armor. So I didn't need to create any new code, I just modified existing objects to essentially create new ones.

bloxgate commented:
So, what then, does the augmented lung do? The Augmented Liver actually does have some bonuses to it, hence the "comparable to" makes me expect the lungs to do stuff as well.

lngredients commented:
Yeah, it's tougher then your average lung, and I was thinking about changing breath-ability and toxin resistance, but I am not familiar enough with how the numbers contained in those variables would effect gameplay to do so. So they are, for now, just tougher lungs.

bloxgate commented:
I'm not sure if those values should be changed through just var-edits. If they're just tougher for now, fair enough then.

bloxgate reviewed (changes requested):
Map contains at least one occurrence of step_x or step_y. These should either be removed or changed to pixel_x/pixel_y as needed.

lngredients commented:
It's essentially the same as changing armor values on the armors I made. When I tested it it all seemed to work.

lngredients commented:
"Map contains at least one occurrence of step_x or step_y. These should either be removed or changed to pixel_x/pixel_y as needed." Not sure what this means.

bloxgate commented:
I was referring to editing their toxin resistance and such, just changing the health should be fine AFAIK.

lngredients commented:
Yeah, like I said, I have no reference to what those would do, so I left those values alone.

bloxgate commented:
When editing the map, you made something appear inside a wall by adjusting its step_x or step_y variables (or hitting the nudge button). These are used for pixel movement, and break movement in game if used in a map.

You should change the apparent location using pixel_x or pixel_y instead.

lngredients committed:
Fixed Map

bloxgate commented:
~~Leave this marked DNM for now. @lngredients mentioned some problems in Discord that need to be sorted first.~~

lngredients committed:
Branch Fix, original files were uploaded directly as a pull request.

DoctorOctopusTheOctopi commented on baystation12.dme:
```diff
@@ -2617,10 +2617,11 @@
 #include "maps\desert_outpost\__desert_outpost_map.dm"
 #include "maps\example\example_define.dm"
-#include "maps\first_contact\all_includes.dm"
-#include "maps\first_contact\mapdef.dm"
```
Why'd you uncheck these two files? If you don't mind me asking

lngredients replied:
They told me to uncheck all the first contact files

lngredients replied:
Should I recheck them?

DoctorOctopusTheOctopi commented on baystation12.dme:
```diff
@@ -2617,10 +2617,11 @@
 #include "maps\geminus_city\geminus_city_map.dm"
+#include "maps\geminus_city\geminus_city_outfits.dm"
 #include "maps\geminus_city\geminus_city_overmap.dm"
+#include "maps\geminus_city\geminus_city_spawns.dm"
```
These includes don't need to be in here, the city/jobs, city/map, city/outfits, and city/spawn.. Uncheck those.. Travis is screaming at it.

[UNIQUENESS: Outfit Datums Shall Have Unique Names]: Names - Marine - The following entries have the same value: /decl/hierarchy/outfit/job/marine and /decl/hierarchy/outfit/job/UNSC_ship/marine
## UNIT_TEST ##: !!! FAILURE !!! [UNIQUENESS: Outfit Datums Shall Have Unique Names]: 1 issue with outfit datums found.
## UNIT_TEST ##: [JOB: Shall have a HUD icon]: Colonist - Missing HUD icon: hudcolonist
## UNIT_TEST ##: [JOB: Shall have a HUD icon]: Colonist - Insurrectionist Sympathiser - Missing HUD icon: hudcolonistinsurrectionistsympathiser
## UNIT_TEST ##: [JOB: Shall have a HUD icon]: Colonist - Insurrectionist Recruiter - Missing HUD icon: hudcolonistinsurrectionistrecruiter
## UNIT_TEST ##: [JOB: Shall have a HUD icon]: Mayor - Missing HUD icon: hudmayor
## UNIT_TEST ##: [JOB: Shall have a HUD icon]: Marine - Missing HUD icon: hudmarine
## UNIT_TEST ##: [JOB: Shall have a HUD icon]: Marine - Squad Leader - Missing HUD icon: hudmarinesquadleader
## UNIT_TEST ##: [JOB: Shall have a HUD icon]: GCPD Officer - Missing HUD icon: hudgcpdofficer
## UNIT_TEST ##: [JOB: Shall have a HUD icon]: Chief of Police - Missing HUD icon: hudchiefofpolice
## UNIT_TEST ##: !!! FAILURE !!! [JOB: Shall have a HUD icon]: icons/mob/hud.dmi - 0 failed sanity checks, 8 jobs with missing HUD icon.

lngredients replied:
Hmm... let me check and see

lngredients replied:
It might have been from the new push I made last night

lngredients replied:
Oh, no, I was told that the overmap and spawns were needed. Shall I uncheck those as well?

DoctorOctopusTheOctopi replied:
I'd just unchange whatever you changed the latest commit. It was fine the second to last commit, but the last commit threw Travis into a loop. Do you know what you did?

lngredients replied:
All I did was copy and paste the map file from my original cloned files (the one I worked on before and submmited via the website) into my repository branch on my local github. The map file is the same one from the very first commit, so I didn't think there should be any issue. How do I undo the last push?

lngredients replied:
That should do it, I reverted the "Branch Fix commit"

lngredients replied:
Try running the check now.

bloxgate replied:
You need to push the revert commit

lngredients replied:
Right, should be pushed now.

lngredients replied:
I accidently reverted the map fix that Blox did, so I re-reverted

lngredients replied:
FML

lngredients replied:
I'll pray to the code gods that that did it. :(

lngredients replied:
Oh hold on I need to uncheck the files I think.

lngredients committed:
Revert "Fixed Map"

This reverts commit 663ea0e8dd135c3f284d0ad14f860f29f3d3fe13.

lngredients committed:
Revert "Branch Fix, original files were uploaded directly as a pull request."

This reverts commit 28ee34a9433500c7ac8802f7a37a7f5bb92a3e53.

lngredients committed:
Revert "Revert "Fixed Map""

This reverts commit 1fe37b43a3c26c480028e890c2377c7ffc465116.

bloxgate commented:
```
git revert 28ee34a9433500c7ac8802f7a37a7f5bb92a3e53
```

DoctorOctopusTheOctopi reviewed (approved):
Looks golden. I can't see anything wrong with the raw .dmm or any items/objects that worry me. Ready for a live test. Author said they playtested. 👍 🐙
